sensors/accelerometer.h

```diff
@@ -0,0 +1,12 @@
+#ifndef ACCELEROMETER_H
+#define ACCELEROMETER_H
+
+class Accelerometer: public Sensor
```

Comment:
I'm sort of confused how you reference `Sensor` without actually including the `Sensor` header or even a forward declaration. Might just be missing something here, but you probably want to `#include` that `Sensor` header.